packages/amplify-provider-awscloudformation/src/display-helpful-urls.js

```diff
@@ -9,6 +11,7 @@ async function displayHelpfulURLs(context, resourcesToBeCreated) {
   showContainerHostingInfo(context, resourcesToBeCreated);
   showHostedUIURLs(context, resourcesToBeCreated);
   await showRekognitionURLS(context, resourcesToBeCreated);
+  await showCognitoSandBoxMessage(context, resourcesToBeCreated);
```

Comment:
My plan is to move all these code into respective category plugin in an upcoming PR. It would be based on and event fired by provider plugin and it would be per operation type like `prePush`, `postPush`, `preStatus` and `postStatus`.